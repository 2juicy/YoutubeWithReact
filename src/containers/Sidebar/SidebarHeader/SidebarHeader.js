import React from "react";
import { Menu } from "semantic-ui-react";
import "./SidebarHeader.scss";

export default function SidebarHeader({ title }) {
  return (
    <Menu.Item>
      <Menu.Header className="sidebar-header">
        {title.toUpperCase()}
      </Menu.Header>
    </Menu.Item>
  );
}
