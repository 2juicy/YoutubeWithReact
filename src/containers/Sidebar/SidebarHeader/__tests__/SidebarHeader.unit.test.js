import React from "react";
import { shallow } from "enzyme";
import SidebarHeader from "../SidebarHeader";

describe("SidebarHeader", () => {
  test("renders", () => {
    const wrapper = shallow(<SidebarHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
