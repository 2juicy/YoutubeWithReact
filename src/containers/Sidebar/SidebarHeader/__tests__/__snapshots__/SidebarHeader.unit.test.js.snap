// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarHeader renders 1`] = `
<MenuItem>
  <MenuHeader
    className="sidebar-header"
  />
</MenuItem>
`;
